ext/fileinfo/fileinfo.stub.php

```diff
@@ -9,23 +9,23 @@ public function __construct(int $flags = FILEINFO_NONE, ?string $magic_database
 
     /**
      * @param resource|null $context
-     * @return string|false
+     * @tentative-return-type
      * @alias finfo_file
      */
-    public function file(string $filename, int $flags = FILEINFO_NONE, $context = null) {}
+    public function file(string $filename, int $flags = FILEINFO_NONE, $context = null): string|false {}
 
     /**
      * @param resource|null $context
-     * @return string|false
+     * @tentative-return-type
      * @alias finfo_buffer
      */
-    public function buffer(string $string, int $flags = FILEINFO_NONE, $context = null) {}
+    public function buffer(string $string, int $flags = FILEINFO_NONE, $context = null): string|false {}
 
     /**
-     * @return bool
+     * @tentative-return-type
      * @alias finfo_set_flags
      */
-    public function set_flags(int $flags) {}
+    public function set_flags(int $flags): bool {}
```

Comment:
Hm, an unfortunate thing about this is that it further enshrines the incorrect return type (which should be void, not bool).

Reply:
Do you see any good solution to circumvent this issue? Or is it better to omit the declaration of those return types which doesn't really make sense?

Reply:
I removed the tentative type from this method, and added a todo